cisco_aci/datadog_checks/cisco_aci/capacity.py

```diff
@@ -0,0 +1,93 @@
+# (C) Datadog, Inc. 2018
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+from . import metrics as aci_metrics
+from . import helpers
+
+
+class Capacity:
+
```

Comment:
can we add a docstring describing what's this class for?

Reply:
yes